components/webui/webui_resources.cc

```diff
@@ -0,0 +1,977 @@
+              {"walletAddress", IDS_BRAVE_REWARDS_INTERNALS_WALLET_ADDRESS},
+              {"walletHistory", IDS_BRAVE_REWARDS_INTERNALS_WALLET_HISTORY},
+              {"walletInfo", IDS_BRAVE_REWARDS_INTERNALS_WALLET_INFO},
+              {"walletNotCreated",
+               IDS_BRAVE_REWARDS_INTERNALS_WALLET_NOT_CREATED},
+              {"walletPaymentId",
+               IDS_BRAVE_REWARDS_INTERNALS_WALLET_PAYMENT_ID},
+              {"walletStatus", IDS_BRAVE_REWARDS_INTERNALS_WALLET_STATUS},
+              {"walletStatusNotConnected",
+               IDS_BRAVE_REWARDS_INTERNALS_WALLET_STATUS_NOT_CONNECTED},
+              {"walletStatusVerified",
+               IDS_BRAVE_REWARDS_INTERNALS_WALLET_STATUS_VERIFIED},
+              {"walletStatusDisconnectedVerified",
+               IDS_BRAVE_REWARDS_INTERNALS_WALLET_STATUS_DISCONNECTED_VERIFIED},
+              {"walletCreationEnvironment",
+               IDS_BRAVE_REWARDS_INTERNALS_WALLET_CREATION_ENVIRONMENT},
+              {"currentEnvironment",
+               IDS_BRAVE_REWARDS_INTERNALS_CURRENT_ENVIRONMENT},
+
+              {"loading", IDS_BRAVE_REWARDS_LOADING_LABEL},
+          }},
+      {
+#if BUILDFLAG(ENABLE_TOR)
+          "tor-internals",
+          {
+              {"tabGeneralInfo", IDS_TOR_INTERNALS_TAB_GENERAL_INFO},
+              {"tabLogs", IDS_TOR_INTERNALS_TAB_LOGS},
+              {"torControlEvents", IDS_TOR_INTERNALS_TOR_CONTROL_EVENTS},
+              {"torVersion", IDS_TOR_INTERNALS_TOR_VERSION},
+              {"torPid", IDS_TOR_INTERNALS_TOR_PID},
+              {"torProxyURI", IDS_TOR_INTERNALS_TOR_PROXY_URI},
+              {"torConnectionStatus", IDS_TOR_INTERNALS_TOR_CONNECTION_STATUS},
+              {"torInitProgress", IDS_TOR_INTERNALS_TOR_INIT_PROGRESS},
+          }},
+      {
+#endif
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
+          "webcompat",
+          {
+              // Report modal
+              {"reportModalTitle",
+               IDS_BRAVE_WEBCOMPATREPORTER_REPORT_MODAL_TITLE},
+              {"reportExplanation",
+               IDS_BRAVE_WEBCOMPATREPORTER_REPORT_EXPLANATION},
+              {"reportDisclaimer",
+               IDS_BRAVE_WEBCOMPATREPORTER_REPORT_DISCLAIMER},
+              {"reportDetails", IDS_BRAVE_WEBCOMPATREPORTER_REPORT_DETAILS},
+              {"reportContactPlaceholder",
+               IDS_BRAVE_WEBCOMPATREPORTER_REPORT_CONTACT_PLACEHOLDER},
+              {"reportContactLabel",
+               IDS_BRAVE_WEBCOMPATREPORTER_REPORT_CONTACT_LABEL},
+              {"cancel", IDS_BRAVE_WEBCOMPATREPORTER_CANCEL},
+              {"submit", IDS_BRAVE_WEBCOMPATREPORTER_SUBMIT},
+              // Confirmation modal
+              {"thankYou", IDS_BRAVE_WEBCOMPATREPORTER_THANK_YOU},
+              {"confirmationNotice",
+               IDS_BRAVE_WEBCOMPATREPORTER_CONFIRMATION_NOTICE},
+          }
+#endif
+      },
+  }};
+
+  return (*kLocalizedStrings)[webui_name];
```

Comment:
this should be `find() != end ? *it : {}`

Reply:
Hmm, if I return `{}` to a `span` isn't it going to be dangling pointer when accessed?

Reply:
> Hmm, if I return `{}` to a `span` isn't it going to be dangling pointer when accessed?

nope, span is just two pointers/iterators. That's why you always return it by value and it can be empty.

Reply:
Fixed. I was worried that returning `{}` to a `base::span` would have a dangling pointer since the `{}` is local, but I realized it's just an initializer list that initializes it to empty, and it's safe.

I did:
```c++
if (auto it = find(); it != end) {
  return it->second;
}
return {};
```